.github/workflows/tests.yml

```
@@ -0,0 +1,40 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+
+
+jobs:
+  test:
+    name: "Go ${{ matrix.go-version }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go-version: [1.14.x, 1.15.x, 1.16.x]
+      fail-fast: true
+
+    steps:
+      - name: Acquire sources
+        uses: actions/checkout@master
+
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+
+      - name: Apply module cache
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Invoke tests
+        run: |
+          go test
```

To get a more verbose output it would make sense to provide the `-v` flag:

```suggestion
          go test -v ./...
```

---

See 191fbc92796. Will it be okay without the `./...`?

---

Yes, that's actually only needed if you want to run tests in a subdirectory, e.g. `./pkg/...`